preset/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
No need for `tsconfig.json` if we don't have any TS files here?

Reply:
Yes, I just added it for this draft, because otherwise commit hooks wouldn't let me push it and I decided to deal with those issues later.